components/views/about/index.tsx

```diff
@@ -1,14 +1,14 @@
-import { StyledAbout } from './About.styles'
+import HeroImage from '@/components/HeroHeader'
 import Headshot from '../../../assets/images/Aj-Headshot.jpg'
+import { StyledPageContainer } from '../PageContainer.styles'
+import { StyledAbout } from './About.styles'
 
 const AboutPage = () => {
     return (
-        <StyledAbout>
-            <div className="hero">
-                <div className="overlay" />
-            </div>
-            <div className="about-container">
-                <div className="content">
+        <StyledPageContainer>
+            <HeroImage url="https://images.unsplash.com/photo-1554224155-3a58922a22c3?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1741&q=80" />
+            <div className="container">
```

Comment:
This looks good. No real comment here, just confirming this is the right path. The next thing to do is to style this .container div inside of the `StyledPageContainer`, and make sure the class exists on every page in the same format